packages/dbos-sqs/README.md

```diff
@@ -80,7 +80,7 @@ The `@SQSConfigure` decorator should be applied at the class level to identify t
 interface SQSConfig {
     awscfgname?: string;
     awscfg?: AWSServiceConfig;
-    queueURL?: string;
+    queueUrl?: string;
```

Comment:
👀

Reply:
Old one is still available, this is the README not the code.

Reply:
What's this? A present for DBOS biggest fanboy?

Reply:
Apparently... ;)